proc_scripts/plotting.py

```diff
@@ -48,12 +48,13 @@ def side_by_side(self,plotname,s,thisrange):
         thisfig,(ax1,ax2) = subplots(1,2)
         self.next(plotname, fig=thisfig)
         sca(ax1)
+        #self.set_units('t2',None)
         forplot = s['t2':expand_limits(thisrange,s)]
-        self.image(forplot)
+        self.image(forplot,human_units=False)
         print("drawing limits",thisrange)
         draw_limits(thisrange,forplot)
         sca(ax2)
-        self.image(forplot.C.cropped_log())
+        self.image(forplot.C.cropped_log(),human_units=False)
```

Comment:
why did you add `human_units=False` here?

Reply:
will not plot if not there.  was a quick fix

Reply:
ok, yes, here this makes sense